.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@6.1.0
+  python: circleci/python@2.1.1
+
+jobs:
+  plan:
+    executor: python/default
+    steps:
+      - checkout
+      - run: 
+          name: Plan Module Updates
+          command: | 
+            pip install PyGithub
```

Comment:
Can we move this to its own step (this will help improve caching/timing)

Reply:
If we are able to put the package in a requirements.txt and then install using `pip3 install -r requirements.txt` that would be ideal. Also check to see if python orb has an install command they may, I'm 60% confident lol.